src/uct/ib/configure.m4

```diff
@@ -418,6 +426,22 @@ AS_IF([test "x$with_ib" = "xyes"],
        AS_IF([test -d "$mlnx_valg_libdir"],
                [AC_MSG_NOTICE([Added $mlnx_valg_libdir to valgrind LD_LIBRARY_PATH])
                valgrind_libpath="$mlnx_valg_libdir:$valgrind_libpath"])
+
+       # EFA device support
+       AS_IF([test "x$with_efa_dv" != xno],
+             [AC_CHECK_HEADER([infiniband/efadv.h], [], [with_efa_dv=no])])
+
+       AS_IF([test "x$with_efa_dv" != xno],
+             [AC_CHECK_LIB([efa], [efadv_query_device],
+                           [AC_SUBST(LIB_EFA, [-lefa])
+                            AC_DEFINE([HAVE_EFA_DV], [1], [EFA device support])],
+                           [with_efa_dv=no], [-libverbs])])
+
+       AS_IF([test "x$with_efa_dv" != xno],
+             [AC_CHECK_DECLS([EFADV_DEVICE_ATTR_CAPS_RDMA_READ],
+                             [AC_DEFINE([HAVE_DECL_EFA_DV_RDMA_READ], [1], [HAVE EFA device with RDMA READ support])],
+                             [], [[#include <infiniband/efadv.h>]])])
```

Comment:
maybe move EFA to separate m4 file (and later on will also move mlx5)?
for now, can just include it from here